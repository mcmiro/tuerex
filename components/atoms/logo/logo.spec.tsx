/* eslint-disable no-use-before-define */
import * as React from 'react';
import { screen, render } from '@testing-library/react';
import Logo from './';

describe('Test input component', () => {
  it('Should pass when name, variant, size and placeholder is additional', () => {
    render(<Logo />);

    const renderInputComponent = screen.getByRole('link');
    expect(renderInputComponent).toBeDefined();
    expect(renderInputComponent).toMatchSnapshot();
  });
});
