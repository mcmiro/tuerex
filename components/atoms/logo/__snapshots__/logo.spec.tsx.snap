// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test input component Should pass when name, variant, size and placeholder is additional 1`] = `
<a
  class="relative h-[28px] block"
  href="/"
>
  <img
    alt="logo"
    data-nimg="1"
    decoding="async"
    height="28"
    loading="lazy"
    src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
    srcset="/_next/image?url=%2Fimg.jpg&w=96&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
    style="color: transparent;"
    width="85"
  />
</a>
`;
